src/styles/base/_base.scss

```diff
@@ -0,0 +1,34 @@
+/**
+ * Reset/normalize
+ */
+
+@import '../abstracts/mixins';
+
+*,
+*::after,
+*::before {
+	margin: 0;
+	padding: 0;
+	box-sizing: inherit;
+}
+
+html {
```

Comment:
Let's use `:root` for higher specification, even though `:root` and `html` are equal in HTML template
https://css-tricks.com/almanac/selectors/r/root/